hostmgr/host.go

```diff
@@ -7,28 +7,27 @@ import (
 	"net"
 	"os"
 	"path"
-	"path/filepath"
-	"strings"
 	"time"
 )
 
 const hostConfFile = "conf.json"
 
 // Host represents a node within the mayu cluster.
 type Host struct {
-	Id               int       `json:",omitempty"`
-	ProviderId       string    `json:",omitempty"`
-	Enabled          bool      `json:",omitempty"`
-	Name             string    `json:",omitempty"`
-	Serial           string    `json:",omitempty"`
-	MacAddresses     []string  `json:",omitempty"`
-	InternalAddr     net.IP    `json:",omitempty"`
-	IPMIAddr         net.IP    `json:",omitempty"`
-	Hostname         string    `json:",omitempty"`
-	MachineID        string    `json:",omitempty"`
-	LastBoot         time.Time `json:",omitempty"`
-	Profile          string    `json:",omitempty"`
-	EtcdClusterToken string    `json:",omitempty"`
+	Id               int               `json:",omitempty"`
+	ProviderId       string            `json:",omitempty"`
+	Enabled          bool              `json:",omitempty"`
+	Name             string            `json:",omitempty"`
+	Serial           string            `json:",omitempty"`
+	MacAddresses     []string          `json:",omitempty"`
+	InternalAddr     net.IP            `json:",omitempty"`
+	AdditionalAddrs  map[string]net.IP `json:",omitempty"`
```

Comment:
adding new entry here `AdditionalAddrs  map[string]net.IP` will be a map with `interface-name` : `assigned IP `